### PR TITLE
Updates and screenshots

### PR DESCRIPTION
**Pull request by SonofUgly**
Kingdom Hearts 3D, Dragon Quest Monsters 1, Bravely Default game issues updated to remove link to geometry shaders issue.
Screenshots added for Theatrhythm, SM3DL, Monkey Ball 3D, and Smash Bros.

### CODE REVIEW

**Comment by Leo626** [comment]
Shouldn't the `date` and `version` be changed to reflect the update in compatibility?

File: games/dragon-quest-monsters-terry-no-wonderland-3d/game.dat [hunk 1]
```diff
@@ -11,7 +10,7 @@ release_date = "2012-05-31"
 
 [[ testcases ]]
 title = "0004000000074600"
-compatibility = "1"
+compatibility = "0"
```
Comment (Schplee) [inline games/dragon-quest-monsters-terry-no-wonderland-3d/game.dat]: As @Leo626 has stated. Compatibility changes should be updated in a new test case.

**Comment by SonofUgly** [comment]
Updated date and version for DQM1.

**Comment by Leo626** [comment]
Since you're removing #1053, you should update your testcase for Bravely Default as well and add your own testcase for Kingdom Hearts.

**Comment by SonofUgly** [comment]
Updated and added testcase.

**Comment by Leo626** [comment]
LGTM 👍

**Pull request merged**